components/mpas-ocean/src/Registry.xml

```diff
@@ -1460,6 +1595,7 @@
 		<package name="tidalPotentialForcingPKG" description="This package controls variables required for tidal potential forcing"/>
 		<package name="topographicWaveDragPKG" description="This package controls variables required for topographic wave drag"/>
 		<package name="gotmPKG" description="This package is for GOTM variables, which are only needed when using GOTM for the vertical turbulence closure."/>
+                <package name="nonHydrostaticMode" description="This package is for variables needed for the nonHydrostatic capabilities"/>
```

Comment:
```suggestion
                <package name="nonhydrostaticPKG" description="This package is for variables needed for the nonHydrostatic capabilities"/>
```
All packages should end in `PKG` for clarity.  I think we want to always use `nonhydrostatic`, not `nonHydrostatic`.